test/images/image-util.sh

```diff
@@ -126,17 +126,17 @@ build() {
     # image tag
     TAG=$(<VERSION)
 
+    # NOTE(claudiub): Some Windows images might require their own Dockerfile
+    # while simpler ones will not. If we're building for Windows, check if
+    # "Dockerfile_windows" exists or not.
+    dockerfile_name="Dockerfile"
+    if [[ "$os_name" = "windows" && -f "Dockerfile_windows" ]]; then
+      dockerfile_name="Dockerfile_windows"
+    fi
+
+    BASEIMAGE=""
```

Comment:
If we're explicitly setting this to empty, the below if block will never get triggered.  Is this intentional?

Reply:
The if below tests the file ``BASEIMAGE``'s existence. Perhaps it would be a good idea to rename this variable to ``baseimage`` to more easily distinguish them.